app/controllers/UploadFileController.scala

```diff
@@ -66,7 +66,7 @@ class UploadFileController @Inject() (
       _                      <- sessionRepository.set(updatedAnswers)
     } yield Ok(view(preparedForm, upscanInitiateResponse)))
       .recover {
-        case _: Exception => throw new UpscanTimeoutException
+        case _: Exception => Redirect(routes.ThereIsAProblemController.onPageLoad())
```

Comment:
@BriWak Can we add an info log message to know the error

Reply:
Added now